src/js/tabs/osd.js

```diff
@@ -1369,6 +1374,87 @@ OSD.loadDisplayFields = function() {
             positionable: true,
             preview: 'AUX',
         },
+        SYS_GOGGLE_VOLTAGE: {
+            name: 'SYS_GOGGLE_VOLTAGE',
+            text: 'osdTextElementSysGoggleVoltage',
+            desc: 'osdDescElementSysGoggleVoltage',
+            defaultPosition: -1,
+            draw_order: 485,
+            positionable: true,
+            preview: 'G 16.8V',
+        },
+        SYS_VTX_VOLTAGE: {
+            name: 'SYS_VTX_VOLTAGE',
+            text: 'osdTextElementSysVtxVoltage',
+            desc: 'osdDescElementSysVtxVoltage',
+            defaultPosition: -1,
+            draw_order: 490,
+            positionable: true,
+            preview: 'A 12.6V',
+        },
+        SYS_BITRATE: {
+            name: 'SYS_BITRATE',
+            text: 'osdTextElementSysBitrate',
+            desc: 'osdDescElementSysBitrate',
+            defaultPosition: -1,
+            draw_order: 495,
+            positionable: true,
+            preview: '50MBPS',
+        },
+        SYS_DELAY: {
+            name: 'SYS_DELAY',
+            text: 'osdTextElementSysDelay',
+            desc: 'osdDescElementSysDelay',
+            defaultPosition: -1,
+            draw_order: 500,
+            positionable: true,
+            preview: '24.5MS',
+        },
+        SYS_DISTANCE: {
+            name: 'SYS_DISTANCE',
+            text: 'osdTextElementSysDistance',
+            desc: 'osdDescElementSysDistance',
+            defaultPosition: -1,
+            draw_order: 505,
+            positionable: true,
+            preview: '10' + FONT.symbol(SYM.METRE),
```

Comment:
Sorry sonar is really picky: 

```suggestion
            preview: `10${FONT.symbol(SYM.METRE)}`,
```

Same on  
/home/runner/work/betaflight-configurator/betaflight-configurator/src/js/tabs/osd.js
Error:   1420:22  error  Unexpected string concatenation  prefer-template
Error:   1429:22  error  Unexpected string concatenation  prefer-template
Error:   1438:22  error  Unexpected string concatenation  prefer-template
Error:   1447:22  error  Unexpected string concatenation  prefer-template

Reply:
Yup, spotted that!

Reply:
Thanks. It's building now!!!

Reply:
@SteveCEvans guess you didn't notice the back ticks here otherwise the ${var} template literal is not expanded

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals